integration_tests/src/main/python/parquet_test.py

```diff
@@ -70,19 +70,29 @@ def read_parquet_sql(data_path):
 original_parquet_file_reader_conf = {'spark.rapids.sql.format.parquet.reader.type': 'PERFILE'}
 multithreaded_parquet_file_reader_conf = {'spark.rapids.sql.format.parquet.reader.type': 'MULTITHREADED'}
 coalesce_parquet_file_reader_conf = {'spark.rapids.sql.format.parquet.reader.type': 'COALESCING'}
+coalesce_parquet_file_reader_multithread_filter_chunked_conf = {'spark.rapids.sql.format.parquet.reader.type': 'COALESCING',
```

Comment:
I don't see any test with different size limits here? Some limit values ranging from small to large should be better than the default max 2GB value.

Reply:
The default limit in these tests is 512 MiB. We don't have any files that are read that would produce more than a single batch anyways. I don't think we have any files that would be larger than a page per row so even if we tried to turn it on the output would be the same. I have done manual tests because the size needed does not really lend itself to this type of test.